experimental/Schemes/RationalMap.jl

```diff
@@ -510,20 +510,29 @@ function pullback(phi::RationalMap, C::AbsAlgebraicCycle)
   R = coefficient_ring(C)
   comps = IdDict{IdealSheaf, elem_type(R)}()
   for I in components(C)
-    # Find a patch in Y on which this component is visible
-    all_V = [V for V in affine_charts(Y) if !isone(I(V))]
-    for V in all_V
-  # @show all_V
-  # min_var = minimum([ngens(OO(V)) for V in all_V])
-  # @show min_var
-  # all_V = [V for V in all_V if ngens(OO(V)) == min_var]
-  # @show all_V
-  # deg_bound = minimum([maximum([total_degree(lifted_numerator(g)) for g in gens(I(V))]) for V in all_V])
-  # @show deg_bound
-  # all_V = [V for V in all_V if minimum([total_degree(lifted_numerator(g)) for g in gens(I(V))]) == deg_bound]
-  # @show all_V
-  # V = first(all_V)
-  # @show findfirst(x->x===V, affine_charts(Y))
+    @show "trying cheap"
+    pbI = _try_pullback_cheap(phi, I)
+    if pbI === nothing
+      @show "trying randomized"
```

Comment:
These verbose printing messages are great. Here is the way to do it properly:
```suggestion
      @vprint :RationalMap 1 "trying randomized pullback\n"
```

Reply:
Then one also has to add an `add_verbose_scope!(:RationalMap)` in our `__init__` in `Oscar.jl`